isis/src/base/apps/cam2cam/cam2cam.xml

```diff
@@ -154,6 +229,295 @@
           </option>
         </list>
       </parameter>
+      
+      <parameter name="OFFBODY">
+        <type>boolean</type>
+        <default><item>FALSE</item></default>
```

Comment:
Similar to the API comment, I think this should be set to true to maintain default behavior.

Reply:
See previous comment for closing